docs/reference/object-model.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: default
+title: Object Model 
+description: The lakeFS object model blends the object models of git and of object stores such as S3.  Here are the explicit definitions.
+parent: Reference
+nav_order: 70
+has_children: false
+---
+
+# Commands (CLI) Reference
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Introduction
+
+lakeFS blends concepts from object stores such as S3 with concepts from Git.  This reference
+defines the common concepts of lakeFS.  Every concept appearing in _italic text_ is its
+definition.
+
+## lakeFS the object store
+
+lakeFS is an object store, and borrows concepts from S3.
+
+An _object store_ links objects to paths.  An _object_ holds:
+
+* Some _contents_, with unlimited size and format.
+* Some _metadata_, including
+  + _size_ in bytes
+  + the _creation time_, a timestamp with seconds resolution
+  + a _checksum_ string which uniquely identifies the contents
+  + some _user metadata_, a small map of strings to strings.
+
+In lakeFS objects are immutable and never rewritten.
+
+A _path_ is a readable string, typically decoded as UTF-8.  lakeFS maps paths to their objects
+according to specific rules.  Importantly (and unlike many object stores), lakeFS may map
+multiple paths to the same object on backing storage.  lakeFS paths use the `lakefs` protocol,
```

Comment:
Again, this section mixes two concepts:
- the backing storage
- lakeFS as an object store

I think we should stick to the second one here, with the first one being explained later.